exercises/etl/canonical-data.json

```diff
@@ -0,0 +1,62 @@
+{
+  "transform": {
+    "description": "transforms the a set of scrabble data previously indexed by the tile score to a set of data indexed by the tile letter",
+    "cases": [
+      {
+        "description": "a single letter",
+        "input": {
+          "1": ["A"]
```

Comment:
I know that this is a `"1":` (rather than `1:`) because JSON would only allow string keys in their objects.

Is it the case that languages will usually use integers as their keys? If so, perhaps a note saying so? Can put it on the same level as `description` and `cases`. No standard key for that note, for some reason people ahve used `"#":` in the past, but maybe `"notes":` is better.